superset/assets/src/explore/exploreUtils.js

```diff
@@ -105,7 +123,11 @@ export function getExploreUrlAndPayload({
 }
 
 export function exportChart(formData, endpointType) {
-  const { url, payload } = getExploreUrlAndPayload({ formData, endpointType });
+  const { url, payload } = getExploreUrlAndPayload({
+    formData,
+    endpointType,
+    allowDomainSharding: false,
```

Comment:
is there any downside to allowing domain sharding for `exportChart`?

Reply:
when user clicks on `explore chart` i don't want subdomains showing up in new window, because user may save the url somewhere, but we may abandon/change subdomains any time.
Same reason for `export json` link.
'export csv' didn't expose URL to user, but it shared function with `export json` feature. so all of them will use original hostname.